### ISSUE TITLE
KubeSphere3.3.1 rerun jenkins pipeline,it's always says "Started by user admin"

### ISSUE THREAD

**Issue by lucumt** [kubesphere/kubesphere#5883]
`KuberSphere` version:
![image](https://github.com/kubesphere/kubesphere/assets/5226038/ec2b330d-2c42-40b6-963f-aa4154b46fca)

**How To Reproduce**
![image](https://github.com/kubesphere/kubesphere/assets/5226038/7249c6bb-5bac-44eb-8b93-4dcd1c76e86e)

**Expected behavior**
It will display the current user who rerun the pipeline,so that we can track the build record by different user exactly.

**Comment by chilianyi** [comment]
Yes, when clicking "replay," it displays actions performed by the admin user. We will address this issue in a future release. Thank you for your feedback.
